### ISSUE TITLE
pav wait occasionally fails

### ISSUE THREAD

**Issue by nicholas-sly** [hpc/pavilion2#276]
`pav wait` command fails on occasion after several iterations, but not constantly, with the below error/traceback.  Restarting the command works, but could fail after some number of iterations as well.

```
Unknown error running command wait: Could not load attributes file:
('Expecting value: line 1 column 1 (char 0)',).
Traceback (most recent call last):
  File "/usr/projects/hpctest/pav2-next/src/lib/pavilion/test_run.py", line 616, in load_attributes
    self._attrs = json.load(attr_file)
  File "/opt/python/3.7.3.2/lib/python3.7/json/__init__.py", line 296, in load
    parse_constant=parse_constant, object_pairs_hook=object_pairs_hook, **kw)
  File "/opt/python/3.7.3.2/lib/python3.7/json/__init__.py", line 348, in loads
    return _default_decoder.decode(s)
  File "/opt/python/3.7.3.2/lib/python3.7/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/opt/python/3.7.3.2/lib/python3.7/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/projects/hpctest/pav2-next/src/bin/pav.py", line 117, in main
    sys.exit(cmd.run(pav_cfg, args))
  File "/usr/projects/hpctest/pav2-next/src/lib/pavilion/plugins/commands/wait.py", line 77, in run
    test_obj = TestRun.load(pav_cfg, test_id)
  File "/usr/projects/hpctest/pav2-next/src/lib/pavilion/test_run.py", line 315, in load
    return TestRun(pav_cfg, config, _id=test_id)
  File "/usr/projects/hpctest/pav2-next/src/lib/pavilion/test_run.py", line 206, in __init__
    self.load_attributes()
  File "/usr/projects/hpctest/pav2-next/src/lib/pavilion/test_run.py", line 621, in load_attributes
    .format(err.args)
pavilion.test_run.TestRunError: Could not load attributes file: 
('Expecting value: line 1 column 1 (char 0)',)
```

**Comment by Paul-Ferrell** [comment]
I remember this bug! I guess I reintroduced it.

**Comment by Paul-Ferrell** [comment]
Fixed by #282
